src/js/utils.js

```diff
@@ -23,8 +23,10 @@ export const requestOptions = {
 
 export function httpGet(fetchMethod, path, onSuccess, onError) {
   httpRequest(fetchMethod, path, requestOptions).then(
-    ({ data, success, headers }) => {
-      success ? onSuccess({ data, headers }) : onError(data)
+    ({ data, success, status, headers }) => {
+      success
+        ? onSuccess({ data, status, headers })
+        : onError({ message: data, status, headers })
     }
   )
 }
```

Comment:
How tempting was it to make `httpGet` behave like the rest of the functions? I'm assuming that the ripple effect was much worse.

Reply:
Too much work